tests/test_harvesters.py

```diff
@@ -138,3 +176,66 @@ def test_copernicus_sentinel_harvester(self):
                                                            username='test', password='TEST')
         self.assertIsInstance(harvester._current_crawler, crawlers.CopernicusOpenSearchAPICrawler)
         self.assertIsInstance(harvester._ingester, ingesters.CopernicusODataIngester)
+
+    def test_osisaf_harvester_extra_excludes(self):
+        """ extra excludes should have passed by the excludes as a list in configuration file.
+        Otherwise, accossiated error must be raised """
+        harvester = harvesters.OSISAFHarvester(urls=[''], max_fetcher_threads=1, max_db_threads=1,
+                                               excludes=['ease', '_sh_polstere', ])
+        self.assertListEqual(list(harvester._current_crawler.excludes),
+                             ['/thredds/', 'http', 'ease', '_sh_polstere'])
+        harvester = harvesters.OSISAFHarvester(urls=[''], max_fetcher_threads=1, max_db_threads=1)
+        self.assertListEqual(list(harvester._current_crawler.excludes), ['/thredds/', 'http', ])
+
+        with self.assertRaises(HarvesterConfigurationError):
+            harvester = harvesters.OSISAFHarvester(urls=[''], max_fetcher_threads=1, max_db_threads=1,
+                                                   excludes='ease')
+
+    def test_extra_excludes_with_no_CLASS_EXCLUDE(self):
+        """ shall return the excludes from the config file """
+        class TestCrawler(crawlers.WebDirectoryCrawler):
+            CLASS_EXCLUDE = None
+
+        class TestHarvester(harvesters.WebDirectoryHarvester):
+            ingester = ingesters.DDXIngester
+            crawler = TestCrawler
+        harvester = TestHarvester(urls=[''], max_fetcher_threads=1, max_db_threads=1,
+                                  excludes=['ease', '_sh_polstere', ])
+        self.assertListEqual(list(harvester._current_crawler.excludes), ['ease', '_sh_polstere'])
```

Comment:
it's already a list, no need to explicitly convert it

Reply:
Done in the next commit.